src/create_db.py

```diff
@@ -14,11 +14,174 @@
+        );"""
+    )
+    print("Created qos_table successfully")
+
+    # TRES Table
+    print("Creating tres_table in DB...")
+    conn.execute(
+        """
+            CREATE TABLE IF NOT EXISTS tres_table (
+                creation_time bigint(20)            NOT NULL,
+                mod_time      bigint(20)  DEFAULT 0 NOT NULL,
+                deleted       tinyint(4)  DEFAULT 0 NOT NULL,
+                id            integer                         PRIMARY KEY AUTOINCREMENT,
+                type          tinytext              NOT NULL,
+                name          tinytext              NOT NULL
+        );"""
+    )
+    print("Created tres_table successfully")
+
+    # Association Hour Usage Table
+    print("Creating assoc_usage_hour_table in DB...")
+    conn.execute(
+        """
+            CREATE TABLE IF NOT EXISTS assoc_usage_hour_table (
+                creation_time bigint(20)            NOT NULL,
+                mod_time      bigint(20)  DEFAULT 0 NOT NULL,
+                deleted       tinyint(4)  DEFAULT 0 NOT NULL,
+                id            int(10),
+                id_tres       int(11)     DEFAULT 1,
+                time_start    bigint(20),
+                alloc_secs    bigint(20)  DEFAULT 0 NOT NULL,
+                PRIMARY KEY (id, id_tres, time_start)
+        );"""
+    )
+    print("Created assoc_usage_hour_table in DB...")
+
+    # Association Day Usage Table
+    print("Creating assoc_usage_day_table in DB...")
+    conn.execute(
+        """
+            CREATE TABLE IF NOT EXISTS assoc_usage_day_table (
+                creation_time bigint(20)            NOT NULL,
+                mod_time      bigint(20)  DEFAULT 0 NOT NULL,
+                deleted       tinyint(4)  DEFAULT 0 NOT NULL,
+                id            int(10),
+                id_tres       int(11)     DEFAULT 1,
+                time_start    bigint(20),
+                alloc_secs    bigint(20)  DEFAULT 0 NOT NULL,
+                PRIMARY KEY (id, id_tres, time_start)
+        );"""
+    )
+    print("Created assoc_usage_day_table in DB...")
+
+    # Association Month Usage Table
+    print("Creating assoc_usage_month_table in DB...")
+    conn.execute(
+        """
+            CREATE TABLE IF NOT EXISTS assoc_usage_month_table (
+                creation_time bigint(20)            NOT NULL,
+                mod_time      bigint(20)  DEFAULT 0 NOT NULL,
+                deleted       tinyint(4)  DEFAULT 0 NOT NULL,
+                id            int(10),
+                id_tres       int(11)     DEFAULT 1,
+                time_start    bigint(20),
+                alloc_secs    bigint(20)  DEFAULT 0 NOT NULL,
+                PRIMARY KEY (id, id_tres, time_start)
+        );"""
+    )
+    print("Created assoc_usage_month_table in DB...")
+
+    # Job Table
+    print("Creating job_table in DB...")
+    conn.execute(
+        """
+            CREATE TABLE IF NOT EXISTS job_table (
```

Comment:
in general with the job table, is there a reason to keep the column names as the slurm names?  Perhaps b/c reporting scripts will initially assume those column names?

Reply:
No reason in particular; it just made it easier for me to see which columns I was copying from. I'd be open to renaming them if they make more sense being named to something different!

Reply:
although we should be flexible b/c things are still changing, I think going with flux names vs slurm names is a good idea